FirebaseMessaging.podspec

```diff
@@ -36,7 +36,9 @@ device, and it is completely free.
     'GCC_C_LANGUAGE_STANDARD' => 'c99',
     'GCC_PREPROCESSOR_DEFINITIONS' =>
       'GPB_USE_PROTOBUF_FRAMEWORK_IMPORTS=1 ' +
-      'FIRMessaging_LIB_VERSION=' + String(s.version)
+      'FIRMessaging_LIB_VERSION=' + String(s.version),
+       # Unit tests do library imports using repo-root relative paths.
+      'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}"',
```

Comment:
`HEADER_SEARCH_PATHS` is not a GCC_PREPROCESSOR_DEFINITIONS. It should be a top-level setting.

Reply:
Oops! Nice catch!